docs/UserGuide.md

```diff
@@ -67,7 +67,7 @@ Here are some commands to get you started:
 
    * `list` : Lists all befriendee and volunteer contacts.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 r/volunteer` : Adds a volunteer named `John Doe` to the Elder Scrolls.
+   * ```add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 r/volunteer``` : Adds a volunteer named `John Doe` to the Elder Scrolls.
```

Comment:
actually i cant seem to replicate #162 , when i copy it it shows the whitespace. 

Can i clarify this is some OS diff or something?

Reply:
okay nvm, realised cause this was because the input does not auto convert newlines to whitespace